src/includes/constants.php

```diff
@@ -58,39 +57,39 @@
 }
 
 // Allow a first class env var to be set that disables .env file loading
-if (!getenv('SS_IGNORE_DOT_ENV')) {
+if (!Environment::getenv('SS_IGNORE_DOT_ENV')) {
```

Comment:
This should probably still be getenv, otherwise it's a bit of a circular dependency, and this var can't be set by .env

Reply:
This behaviour is required in case user-specific bootstrap files set environment vars.

Note that `Environment::getenv` doesn't actually require `.env` files to be loaded; It also acts as an accessor to the underlying system environment. This just maintains consistency between all getenv accessors.

Reply:
Fair enough